eng/Versions.props

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MicrosoftFrameworkUdkPackageVersion>1.3.0-CI-25260.1001.221205-1000.0</MicrosoftFrameworkUdkPackageVersion>
     <MicrosoftWindowsCsWinRTPackageVersion>2.0.0</MicrosoftWindowsCsWinRTPackageVersion>
     <CsWinRTDependencyDotNetCoreSdkPackageVersion>6.0.301</CsWinRTDependencyDotNetCoreSdkPackageVersion>
     <CsWinRTDependencyDotNetCoreRuntimePackageVersion>6.0.6</CsWinRTDependencyDotNetCoreRuntimePackageVersion>
     <CsWinRTDependencyWindowsSdkVersionSuffixPackageVersion>27</CsWinRTDependencyWindowsSdkVersionSuffixPackageVersion>
+
     <!-- TODO: Fix DownloadDotNetCoreSdk.ps1 script so we don't need this property -->
     <CsWinRTDependencyDotNetCoreSdkLkgPackageVersion>$(CsWinRTDependencyDotNetCoreSdkPackageVersion)</CsWinRTDependencyDotNetCoreSdkLkgPackageVersion>
+
+    <!-- TODO: REMOVE This belongs in Version.Details.xml -->
```

Comment:
Because of how Maestro updates both version.props and version.details.xml, MicrosoftFrameworkUdkPackageVersion belongs in this file.
I would add it back to line 4.

Reply:
We shouldn't have Versions.props. Let's talk offline

Reply:
Updated as discussed. Also fixed MRT's logic (not merely 'TODO...' but MRT's Lkg logic was simply broken)

See `eng\Version.Details.xml` for docs